deepchecks/vision/checks/train_test_validation/image_property_drift.py

```diff
@@ -193,6 +197,7 @@ def compute(self, context: Context) -> CheckResult:
                     column_type=get_column_type(single_property['output_type']),
                     margin_quantile_filter=self.margin_quantile_filter,
                     max_num_categories_for_drift=self.max_num_categories_for_drift,
+                    min_category_size_ratio=self.min_category_size_ratio,
```

Comment:
Ok, I have the pettiest comment of all comments - but can you reorder the parameters (here and in other places) so the order will be:
min_category_size_ratio
max_num_categories_for_drift
max_num_categories_for_display
?

Makes more sense to put the "max_num_categories" parameters together, and in documentation as well, so users can understand the difference between them more easily